lib/actions.ts

```diff
@@ -47,7 +47,7 @@ export const cancelSubscription = async (subscriptionId: string) => {
   }
 
   const connectSession: ConnectSessionParams = {
-    callbackUrl: 'http://localhost:3000?error=connectSessionFailed',
```

Comment:
(Commenting on this one but it affects all `callbackUrl`s in this file:

Does redirecting to `localhost` explicitly work with the hosted version of this example?

Reply:
I'm not 100% sure I understand the question. Can you elaborate?

Reply:
`localhost` doesn't work with the hosted version, so `callbackUrl` should ideally take current location into account instead of hardcoding a string 🤔

Reply:
Yep, it's not working right now, which is why I mentioned that we need to fix callbackUrl as soon as possible when we talked about this in standup (or planning, don't remember exactly)

Reply:
cool; will update

Reply:
updated; since the connect session was created in server actions, I used the headers to build the current URL